index.bs

```diff
@@ -8353,7 +8455,7 @@ JavaScript code.
 
     <pre highlight="webidl">
         interface I {
-          Promise&lt;void> delay(unrestricted double ms);
+          Promise&lt;void> delay(unrestricted f64 ms);
```

Comment:
Note that I had to revert this due to `unrestricted f64` currently being a syntax error.

Reply:
Would this mean the PR should wait for parser to be updated first?

Reply:
We could probably merge this as‑is, since this PR also defines "aliases" for:

```webidl
// Not actually implemented using `typedef`s, this is just for illustration:
typedef (i8) byte;
typedef (i16) short;
typedef (i32) long;
typedef (i64) long long;

typedef (u8) octet;
typedef (u16) unsigned short;
typedef (u32) unsigned long;
typedef (u64) unsigned long long;

typedef (f32) float;
typedef (unrestricted f32) unrestricted float;
typedef (f64) double;
typedef (unrestricted f64) unrestricted double;
```

Reply:
AFAIK an identifier can't contain a whitespace and `typedef (unrestricted f32) unrestricted float;` is invalid syntax...

Reply:
That’s why this isn’t actually implemented using `typedef`s, that is just for illustration, the actual aliases are defined as:
> For legacy compatibility, `byte` is to be treated as an alias for `i8`, similar to:

Reply:
Ah, missed that part, sorry 😅